TRANSLATING.md

```diff
@@ -13,7 +13,8 @@ Note that we take translations very seriously.
 They should receive as much care as the English version.
 
 For example, Google Translate is already built into Google Chrome.
-Don’t bother to use software to create translations of the docs.
+Don’t bother to create a mere machine translation of the docs.
+You can use machine translations to help you get done faster, but everything needs to be proof-read by you, and typically, a lot of things need to be corrected or rephrased.
```

Comment:
Idk, looks like "to see use your eye"...

Reply:
Is this better?
```suggestion
(You can use machine translations in the process to help you get done faster.)
```